src/components/info_block.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import propTypes from 'prop-types'
+import styled from "styled-components"
+
+import Button from "./button"
+
+const InfoHeading = styled.h2`
+/* Always use uppercase text */
+text-transform: uppercase;
+font-weight: bold;
+
+&::before {
+    content: "×"; 
+    color: var(--MHP-purple);
+    padding-right: 10px;
+    }
+`
+
+const InfoBlock = ({ heading, description, buttonText, image, href, reverseOrder }) => (
+
+    // col order-first and col order-last sets the order of the components
+    <div className="row my-3">
+
+        {/* Text component */}
+        <div className={`col-md ${reverseOrder ? "order-last" : ""}`} >
+
+            <InfoHeading> {heading} </InfoHeading>
+            <p> {description} </p>
+
+            <div className="row">
+                <div className="col-md-8">
+                    <Button href={href}> {buttonText} </Button>
+                </div>
+            </div>
+
+        </div>
+
+        {/* Image component */}
+        <div className="col-md" style={{ height: 500, border: "red solid 2px" }}>
```

Comment:
I don't really know how css works so maybe it'll be fine when we have actual images, but for now the borders seem to be too wide, not in line with the content above/below (as I assume would be intended). Will this be an issue when we have images?

Reply:
These components are still a work in progress. The red border is just a placeholder for an image.